### PR TITLE
install iproute since the origin-base image dose not contain ip command

### PR DESCRIPTION
**Pull request by bmeng**
@danwinship PTAL

### CODE REVIEW

File: images/router/egress/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 FROM openshift/origin-base
 
+RUN yum install -y iproute
```
Comment (danwinship) [inline images/router/egress/Dockerfile]: add "&& yum clean all" for consistency with other Dockerfiles

**Comment by danwinship** [comment]
Sorry, I originally had that there and then convinced myself it wasn't needed somehow...

**Comment by bmeng** [comment]
@danwinship Thanks. Updated.

**Comment by danwinship** [comment]
LGTM. @eparis can you merge it?

**Comment by eparis** [comment]
[merge]

**Pull request merged**